packages/components/src/Tree/TreeCollection.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { CompatibleHTMLProps } from '@looker/design-tokens'
+import React from 'react'
+import styled from 'styled-components'
+
+type TreeCollectionProps = CompatibleHTMLProps<HTMLUListElement>
+
+const TreeCollectionLayout = ({ children, ...props }: TreeCollectionProps) => {
+  return (
+    <ul {...props} role="tree">
+      {children}
+    </ul>
+  )
+}
+
+export const TreeCollection = styled(TreeCollectionLayout)`
+  margin: 0;
+  padding: 0;
+`
```

Comment:
Needs test coverage :)

This can be consolidated into a single component (at least for now).

```suggestion
export const TreeCollection = styled.ul.attrs(({ role = 'tree' }) => ({
  role,
}))<TreeCollectionProps>`
  margin: 0;
  padding: 0;
`
```

Reply:
👍  Added a test file for `TreeCollection` in 8aa464247

It only checks that children are rendered by `TreeCollection` since it's just a styled `ul` at the moment, but we can add tests for `Tree` windowing behavior once that's implemented.